### PR TITLE
Test stoiip function

### PR DESCRIPTION
**Pull request by FreddyEcu-Ch**
Test stoiip function with single variables PEX-15

### CODE REVIEW

File: tests/test_stoiip.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import pytest
+from stoiip_prob import stoiip as stp
+
+
+def test_stoiip():
```
Comment (zeratul-3011) [inline tests/test_stoiip.py:5]: Here we need to add a brief documentation on what is the purpose of this test. You can use the triple quotes with a single message (no need to use all the template that is fill out automatically by PyCharm in this case)

File: tests/test_stoiip.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import pytest
+from stoiip_prob import stoiip as stp
```
[the remark this answers is not in the document]
Comment (FreddyEcu-Ch) [reply tests/test_stoiip.py]: It was done

File: tests/test_stoiip.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+
+def test_stoiip():
+    sto = stp.stoiip(250, 40, 0.20, 0.20, 1.24)
+    assert sto == 10010322.6
```
[the remark this answers is not in the document]
Comment (FreddyEcu-Ch) [reply tests/test_stoiip.py]: Done

File: tests/test_stoiip.py (hunk 1 above)
Comment (FreddyEcu-Ch) [reply tests/test_stoiip.py:5]: Done. 

I activated the docstrings with the NumPy configuration (see screenshot), however, it seems like it is not working in my Pycharm.

Comment (FreddyEcu-Ch) [reply tests/test_stoiip.py:5]: image

**Pull request merged**